build-python-package/action.yml

```
@@ -8,10 +8,17 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Install dependencies
-      shell: bash
-      run: python3 -m pip install wheel
-
     - name: Build
       shell: bash
-      run: python3 setup.py bdist_wheel sdist
+      run: |
+        if [[ -f setup.py ]]; then
+          # old style setup.py project
+          # (must manually install build dependencies)
+          python3 -m pip install wheel
+          python3 setup.py bdist_wheel sdist
```

Was trying to figure out why the test was failing - would help to log this
```suggestion
          echo "setup.py detected"
          python3 -m pip install wheel
          python3 setup.py bdist_wheel sdist
```